graphene_django/conftest.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from graphene_django.settings import graphene_settings
+
+from .registry import reset_global_registry
+
+
+@pytest.fixture(autouse=True)
+def reset_registry_fixture(db):
+    yield None
+    reset_global_registry()
+
+
+@pytest.fixture()
+def settings():
```

Comment:
```suggestion
def graphene_settings():
```

Otherwise this will clash with the pytest django settings fixture: https://pytest-django.readthedocs.io/en/latest/helpers.html#settings